libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1572,8 +1579,16 @@ void WaveTrack::ClearAndPasteOne(
                if (split.right)
                   // new clip was cleared left
                   attachLeft(*newClip, *split.right);
-               bool success = track.AddClip(std::move(newClip));
-               assert(success); // copied clip has same width and factory
+               assert(clip->GetPlayDuration() > 0);
```

Comment:
These three lines would be better placed immediately after the call to `ClearRight`: it took me some time to check that it had nothing to do with what's currently in between.